gulpfile.js

```diff
@@ -1,127 +1,124 @@
+/**
+ * Import libraries
+ */
 const path = require('path');
-
 const gulp = require('gulp');
-const shell = require('gulp-shell');
+const url = require('url');
+const _ = require('lodash');
 
-// `rc` allows all config options to be overridden with CLI flags like `--js.enabled=''` or in `~/.p2-theme-corerc` files, among many others: https://www.npmjs.com/package/rc
-const config = require('rc')('p2-theme-core', require('./gulpconfig.js'));
-const themeCore = require('p2-theme-core');
+/**
+ * Twig namespaces work
+ */
+const namespaceTask = require('./tools/tasks/twig-namespaces');
+namespaceTask.twigNamespaces(gulp);
```

Comment:
This will be addressed in next branch to no longer pass to gulp and provide magic tasks.